src/main/resources/assets/codeclient/lang/en_us.json

```diff
@@ -278,5 +280,11 @@
   "codeclient.command.dfgive.count_above_max": "Maximum stack size of %s is %s",
   "codeclient.command.dfgive.count_below_min": "Minimum stack size is 1",
   "codeclient.command.dfgive.clipboard_unavailable": "Unable to access the clipboard",
-  "codeclient.command.dfgive.clipboard_invalid": "Please copy a valid give command to your clipboard"
+  "codeclient.command.dfgive.clipboard_invalid": "Please copy a valid give command to your clipboard",
+  "codeclient.command.uuid.not_found": "Player %s was not found",
+  "codeclient.command.uuid": "UUID of %s is ",
+  "codeclient.command.uuid.own": "Your UUID is ",
+  "codeclient.command.name.not_found": "UUID %s was not found! Please check if you misspelled it and try again",
+  "codeclient.command.name": "Name of %s is ",
+  "codeclient.command.name.own": "The UUID is yours! What a surprise!"
```

Comment:
uhh

Reply:
I did not know how else I could keep it translatable but underline uuid/name in the message

Reply:
i was talking about the name.own one

Reply:
yeah so? its a funny little easter egg B)